Description: Fixed make File to delete binary 
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 breakmywork (1.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Ravikiran Janardhana <ravikiran.j.127@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- breakmywork-1.0.orig/Makefile
+++ breakmywork-1.0/Makefile
@@ -9,6 +9,7 @@ install:
 		mkdir -p $(IMGDIR)
 		cp breakmywork.py $(BASEDIR)
 		cp images/breakicon.png $(IMGDIR)
-		ln -s $(BASEDIR)/breakmywork.py $(BINDIR)
+		ln -s $(BASEDIR)/breakmywork.py $(BINDIR)/breakmywork
 uninstall:
 		rm -rf $(BASEDIR)
+		rm $(BINDIR)/breakmywork
